src/Components/Recommendation/Recommendation.spec.ct.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { Intl } from '../../Utilities/intlHelper';
+import getStore from '../../Store';
+import _ from 'lodash';
+
+import { Recommendation } from './Recommendation';
+import rule from '../../../cypress/fixtures/api/insights-results-aggregator/v2/rule/external.rules.rule|ERROR_KEY.json';
+import ack from '../../../cypress/fixtures/api/insights-results-aggregator/v2/ack/external.rules.rule|ERROR_KEY.json';
+import clusterDetails from '../../../cypress/fixtures/api/insights-results-aggregator/v2/rule/external.rules.rule|ERROR_KEY/clusters_detail.json';
+import { TOTAL_RISK, CATEGORIES } from '../../../cypress/utils/globals';
+
+const defaultPropsRule = {
+  isError: false,
+  isUninitialized: false,
+  isLoading: false,
+  isFetching: false,
+  isSuccess: true,
+  data: rule,
+  refetch: undefined,
+};
+
+const defaultPropsAck = {
+  data: ack,
+  isFetching: false,
+  refetch: undefined,
+};
+
+const defaultPropsClusterDetails = {
+  data: clusterDetails.data,
+  isFetching: false,
+  isSuccess: true,
+  refetch: undefined,
+};
+
+const ruleDescription = rule.content.description;
+
+const disabledRule = _.cloneDeep(rule);
+disabledRule.content.disabled = true;
+
+describe('recommendation page for enabled recommendation with clusters enabled and disabled', () => {
+  beforeEach(() => {
+    mount(
+      <MemoryRouter>
+        <Intl>
+          <Provider store={getStore()}>
+            <Recommendation
+              rule={{ ...defaultPropsRule }}
+              ack={{ ...defaultPropsAck, data: undefined }}
+              clusters={{ ...defaultPropsClusterDetails }}
+              match={{ params: { recommendationId: 'X' } }}
+            />
+          </Provider>
+        </Intl>
+      </MemoryRouter>
+    );
+  });
+
+  it('header shows description', () => {
+    cy.ouiaType('PF4/Title', 'h1')
+      .should(($el) => expect($el.text().trim()).to.equal(ruleDescription))
+      .and('have.length', 1);
+  });
+
+  it('header shows total risk', () => {
+    // TODO
```

Comment:
Oh, nice, thanks. However, if this is already tested somewhere else, maybe we can just check that the `RuleDetails` component is rendered (I forgot it was tested in another module). What do you think?

Reply:
@ikerreyes , yes, you can check if `.ins-c-rule-details` container is rendered: https://github.com/RedHatInsights/frontend-components/blob/master/packages/advisor-components/src/RuleDetails/RuleDetails.tsx#L75

Reply:
Has this component already implemented and used? I guess JS is doing something in way I do not understand, because I cannot find `.ins-c-rule-details`. Or maybe it needs to be updated with the latests version of frontend-components?
Searching for it also in `stage/beta`, only finds one with `ins-c-rule-details__stack`.

Reply:
this is weird, the class name must be present. seems like it is ignored by PF's `Flex` component. I'll open a bug. can you stick to another selector?

Reply:
Sure, no problem. Thanks